src/util/exportRequestBuilders.ts

```diff
@@ -0,0 +1,34 @@
+import { SupportedExportTypes } from '@/components/query-selector/export-type';
+
+/**
+ * Parameters to build export request string. Will need to add more fields in the future
+ * for element parameters and type filter parameters.
+ */
+export interface BuilderRequest {
+  exportType: SupportedExportTypes;
+  id?: string | null;
+  typeParams: string[];
+  elementParams?: string[]; // may not be string[] in the future
+  typeFilterParams?: string[]; // may not be string[] in the future
+}
+
+/**
+ * Builds bulk export request string from a list of parameters, a SupportedExportType, and an id.
+ * Note: currently only supports type parameters.
+ */
+export function buildExportRequestString(request: BuilderRequest) {
+  const { exportType, id, typeParams } = request;
+  const baseUrl = `${process.env.NEXT_PUBLIC_HOST}:${process.env.NEXT_PUBLIC_PORT}`;
+  const queryString = typeParams.length === 0 ? '' : `?_type=${typeParams.toString()}`;
```

Comment:
```suggestion
  const queryString = typeParams.length !== 0 ?? `?_type=${typeParams.toString()}`;
```
I think? Maybe test rq

Reply:
eslint doesn't like this `Unexpected constant nullishness on the left-hand side of a ??  expression.eslint`[no-constant-binary-expression](https://eslint.org/docs/latest/rules/no-constant-binary-expression). I am not sure if there is a better way. This function will probably change when we have more parameters. I might be able to do something nicer then, like selectively call the function to add type parameters if they exist?

Reply:
Ah nah no worries then! Not worth fussing over, this works! Thanks for checking.